### ISSUE TITLE
Dracula and Nord Dark Color Scheme's cursors incredibly difficult to find.

### ISSUE THREAD

**Issue by namtrah** [kepano/obsidian-minimal#258]
I confirm to have installed and updated the following:
- Minimal Theme
- Minimal Theme Settings plugin
- Contextual Typography plugin

**Confirm that the bug is not present when Minimal theme is off**

This is not happening when the Minimal theme is off.
**Describe the bug**
Using Dark Mode and the Minimal "Dark Mode Color Scheme"(s) of either Nord or Dracula, the cursor is illegible. It is a dark purple on the dark screen and impossible to find if on my computer.

**To reproduce**
Turn on Nord or Dracula Dark Mode Color Scheme and try to find the cursor position.

**Expected behavior**
I would expect, like in the other color mode schemes that the cursor should be easily visible in order to begin writing without first finding where one is.

**Screenshots**
I cannot seem to grab it as it blinks, but it should be easily recreatable.
 
**Debug info (required):**
None happens as this is not truly a bug, but a theme setting issue. I would also not make such a fuss about this, but I cannot find where in Minimal or the Theme Settings where I could change my cursor color differently.

**Comment by namtrah** [comment]
I would like to add that I just found the option of a "Colorful Cursor" which was by default on. Turning this off, does allow me to find the cursor. It is just a shame that it is not configurable to a color I would like and see.

If that is good enough then I am fine to close this. But if you could add a color chooser for the cursor itself, then I'd consider this a feature instead of a bug.

**Comment by kepano** [comment]
I am not sure I am seeing what you are seeing. Here's what I see for the cursor in Nord and Dracula

Nord
<img width="318" alt="Screen Shot 2022-04-02 at 8 33 59 AM" src="https://user-images.githubusercontent.com/10565871/161390318-d82e993c-e998-4a1e-be49-e20a6ddb8d91.png">

Dracula
<img width="287" alt="Screen Shot 2022-04-02 at 8 33 51 AM" src="https://user-images.githubusercontent.com/10565871/161390317-f1d502b4-2a8d-4b56-a4a1-fc540e1168ad.png">

**Comment by kepano** [comment]
This color can be modified in Style Settings > Interface colors > Accent color

**Comment by namtrah** [comment]
Thanks for the hint. So with Divide and Conquer, I disabled all my plugins and still had the issue. I then reinstalled and reenabled only your plugins, still the same...Finally I reset all your settings. Each one. Finally it looked like your pictures. I can say for whatever reason, someone (me) or some plugin (absolve) changed a setting without me realizing and I couldn't get out of it.

Now it is working as described. Thank you for your assistance. I will close this.
